worker/src/python/triton_distributed/worker/deployment.py

```diff
@@ -103,6 +105,9 @@ def start(self):
             if not worker_config.log_level:
                 worker_config.log_level = self._default_log_level
 
+            if self._force_consolidated_logs:
+                worker_config.consolidated_logs = True
```

Comment:
nit: this is where I do wonder - if the name should just be the same between the two - ie
```suggestion
                worker_config.consolidate_logs = self._consolidate_logs || worker_config.consolidate_logs
```